addons/website_slides/models/slide_question.py

```diff
@@ -44,8 +41,9 @@ def _check_at_least_2_answers(self):
 class SlideAnswer(models.Model):
     _name = "slide.answer"
     _rec_name = "text_value"
-    _description = """Answer for a slide question"""
+    _description = "Answer for a slide question"
+    _order = 'question_id, id'
 
-    question_id = fields.Many2one('slide.question', string="Question")
-    text_value = fields.Char("Answer", required=True,)
-    is_correct = fields.Boolean("Is correct answer", default=False)
+    question_id = fields.Many2one('slide.question', string="Question", required=True)
```

Comment:
@tde-banana-odoo  Shouldn't the field be as `ondelete='cascade'` ? Otherwise, we have no way to delete the answers and thus, the questions

Reply:
Probably, indeed.